docs/customer-interactive-api.md

```diff
@@ -0,0 +1,60 @@
+# Livery Video Interactive API
+
+This document describes the API calls that are available to customers. These calls can be used by application administrators to retrieve restricted data.
+
+For more information regarding domain & paths, see: **Livery Video Customer API**
```

Comment:
The domains and paths in the `Livery Video Customer API` are different from the ones in the interactive API.
So I think we should not reference the video documentation for this, but just state what we want to tell the readers.
Since we don't have a single production cluster for interactive but each customer has its own cluster for now we can say that they have to ask for the url with their Livery contact.